backend/src/main/java/de/neuefische/backend/service/ItemService.java

```diff
@@ -14,23 +14,29 @@ public class ItemService {
     private final ItemRepository itemRepository;
 
     public List<Item> getAll() {
-        return itemRepository.findAll();
+        return itemRepository
+                .findAll();
     }
 
     public Item getById(String id) throws Exception {
-        return itemRepository.findById(id).orElseThrow(() -> new Exception("Item with id " + id + " is nonexistent."));
+        return itemRepository
+                .findById(id)
+                .orElseThrow(() -> new Exception("Item with id " + id + " is nonexistent."));
```

Comment:
Would create a custom exception for this

Reply:
Wenn du Zeit hast, kannst du mir das kurz zeigen. Du meintest doch, das geht schnell und einfach?